gcp/bedrock-demos/cloudrun/pocket-demo.env.yaml

```diff
@@ -0,0 +1,22 @@
+# This file contains env vars which will ONLY be used on Mozorg Demos on GCP.
+#
+# This file is public: do _not_ put secrets such as API keys into this file.
+# If you need to add a secret, please consult with SRE or MEAO backend to
+# learn how to do this.
+
+ALLOWED_HOSTS: .tekcopteg.org,.moz.works,.run.app
+CONTENT_CARDS_URL: https://www-dev.allizom.org/media/
+CSP_DEFAULT_SRC: "*.tekcopteg.com"
+CSP_EXTRA_FRAME_SRC: "*.mozaws.net,o1069899.sentry.io"
+CSP_REPORT_ENABLE: "True"
+DB_DOWNLOAD_IGNORE_GIT: "True"
+DEBUG: "False"
+DEV: "True"
+GTM_CONTAINER_ID: GTM-MW3R8V
+LOG_LEVEL: INFO
+PROD_DETAILS_STORAGE: product_details.storage.PDDatabaseStorage
+RUN_SUPERVISOR: "True"
+SECURE_SSL_REDIRECT: "True"
+SENTRY_DSN: https://97ec0cd426714b728e92f3b3aa62f00b@o1069899.ingest.sentry.io/6260338
```

Comment:
Same question as above for Sentry?

Reply:
We currently have a generic "demos" sentry proj. We could split that too if there's enough value